ciacco/miners/mailquotalimit-list

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/perl
+
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - info@nethesis.it
+#
+# This script is part of Dante.
+#
+# Dante is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# Dante is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Dante.  If not, see COPYING.
+#
+
+# List mailbox by quota usage
+
+use strict;
+use warnings;
+use esmith::ConfigDB;
+use JSON;
+
+my @data;
+my $cdb = esmith::ConfigDB->open_ro();
+my $quota_status = $cdb->get_prop('dovecot', 'QuotaStatus') || 'disabled';
+if ($quota_status eq 'enabled' && -f '/usr/libexec/nethserver/mail-quota') {
+    my $quota = decode_json(`/usr/libexec/nethserver/mail-quota`);
+    foreach (keys %$quota) {
+        my $size = int($quota->{$_}{'size'} || 0);
+        my $max = int($quota->{$_}{'max'} || 0);
+        next if ($size == 0);
+        next if ($max == 0);
+        my $quotaLimit = int(($size/$max)*100);
+        push(@data, {name => $_, count => $quotaLimit});
```

Comment:
The script already return a `perc` field, we can use it.

Output example:
```
  "giacomo@nethesis.it": {
    "perc": 9,
    "msg": 114453,
    "max": 20971520,
    "size": 2062054
  },

```

Reply:
good, done